tests/testthat/test-hello_admiral.R

```diff
@@ -0,0 +1,28 @@
+
+S = "Welcome to Admiral family"
```

Comment:
We are trying to move data into test_that calls so there is no spillover issues between test files.

Reply:
Made this change.